src/kernels/miopen_type_traits.hpp

```diff
@@ -76,7 +76,7 @@ struct remove_cv
     typedef typename remove_volatile<typename remove_const<T>::type>::type type;
 };
 
-#if HIP_PACKAGE_VERSION_FLAT >= 6001000000ULL
+#if HIP_PACKAGE_VERSION_FLAT >= 6001024000ULL
```

Comment:
@atamazov now with the <limits> header issue resolved, we found more issues with
```
/tmp/comgr-4e3a20/include/miopen_type_traits.hpp:81:8: error: redefinition of 'integral_constant'
   81 | struct integral_constant
      |        ^
/long_pathname_so_that_rpms_can_package_the_debug_info/src/external/clr/hipamd/include/hip/amd_detail/amd_hip_vector_types.h:55:38: note: previous definition is here
   55 | template <class _Tp, _Tp __v> struct integral_constant {
      |                                      ^

```
I guess we have to identify another point where this workaround is needed, and *not the same point* with <limits>.

Reply:
We can make it
> #if HIP_PACKAGE_VERSION_FLAT > 6001024000ULL

but this is very fragile ...